installation.md

```diff
@@ -34,7 +34,7 @@ First, download the Laravel installer using Composer:
 
     composer global require "laravel/installer"
 
-Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system. For Windows check '~/AppData/Roaming/Composer/vendor/bin' for the 'laravel' executable.
```

Comment:
`~` doesn't mean home profile on Windows. The correct is `%APPDATA%\Composer\vendor`